ecowitt2mqtt/hass.py

```diff
@@ -77,6 +77,10 @@
     UNIT_SYSTEM_METRIC,
 )
 
+# todo -> turn this into a lookup table based on 'model' value?
```

Comment:
I agree that a lookup table is probably our only option (since I can't think of or discover a way to, say, look this info up via an Ecowitt API...). We should also have:

1. A default name and manufacturer
2. In the case of a name/manufacturer that we don't have a mapping for, an `INFO`-level log message instructing the user to report it here

Thoughts?

Reply:
As I'm not a python native, not sure if my implementation is how it's supposed to be done. One thing we currently don't check for is if the model is not provided in the payload. That could potentially cause some issues?